### PR TITLE
Case Study 2

### PR DESCRIPTION
**Pull request by bsouvignier**
I think this has all my Case Study 2 pages included... I hope!

### CODE REVIEW

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <a href="activities.html">Activities</a>
+            <a href="reservations.html">Reservations</a>        
+    </nav>
+    
+    <!-- Main content area -->
+    <div id="content">
+        <h2>Activities</h2>
+        <img src="images/coast.jpg" alt="Beautiful Image of the Pacific Coast" height="250" width="320">
+        <p>
+        Actually there is nothing to do here but jump off a cliff.
```
Comment (htcMosman) [inline activities.html]: Really?  Jump off a cliff huh?

**Comment by bsouvignier** [comment]
Pull request ptCase3

File: pacific.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+h1 {
+    font-family: Georgia,Times,"Times New Roman",serif ;   
+    margin-bottom: 0px;
+    padding: 10px;
+    
+}
+h2 {
+    color: #3399CC;
+    font-family: Georgia,Times,"Times New Roman",serif;
+
```
Comment (htcMosman) [inline pacific.css]: Missing the text shadow on the h2.

File: activities.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        </li>
+        <li><a href="reservations.html">Reservations</a>
+        </li>
+    </ul>
+</nav>
+        <!-- Main content area -->
+        <div id="content">
+            <h2>The Activities at Pacific Trails</h2>
+            <img src="images/trail.jpg" alt="Beautiful Image of a hiking trail">
+            <h3>Hiking</h3>
```
Comment (htcMosman) [inline activities.html]: :-)  The dropbox files still say there's nothing to do but jump off a cliff.  Glad too see we've found *lively* things to do here.

**Pull request closed without merging**